src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosDbCollectionPhysicalPartitionInfo.cs

```diff
@@ -33,19 +35,23 @@ internal class CosmosDbCollectionPhysicalPartitionInfo : IRequireInitializationO
         private readonly IHttpClientFactory _httpClientFactory;
         private readonly ILogger<CosmosDbCollectionPhysicalPartitionInfo> _logger;
         private readonly CancellationTokenSource _backgroundLoopCancellationTokenSource = new();
+        private readonly IAccessTokenProvider _aadTokenProvider;
         private Task _backgroundLoopTask;
 
         public CosmosDbCollectionPhysicalPartitionInfo(
+            IAccessTokenProvider aadTokenProvider,
```

Comment:
What "add" in "aadTokenProvider" stands for?

Reply:
@abiisnn "aad" stands here for Azure Active Directory.